src/assets/styles/base.scss

```diff
@@ -29,25 +29,33 @@
        url('../fonts/BCSans/BCSans-BoldItalic.woff') format('woff'); /* Optimized for very modern browsers */
 }
 
-html,
-body {
-  min-height: 100vh;
+.font-11 {
+  font-size: $px-11;
```

Comment:
This class can be used anywhere in this app, but it can only be `@extend`ed where it's visible, ie, in this file only (since no components import base.scss).
So, I also created font size variables that are used in the components - see theme.scss.

Reply:
What I mean is, some components make use of this class, while other components use the variables instead.